common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -66,6 +66,13 @@ class SequenceFields(object):
         scope=Scope.settings,
     )
 
+    self_paced = Boolean(
+        display_name=_("Course is self-paced"),
+        help=_("ask Eric to remove this field"),
+        default=False,
+        scope=Scope.settings,
+    )
```

Comment:
I believe the problem you are facing is in the way the unit tests are set up.  You shouldn't need this here.